edfi-repo-auditor/edfi_repo_auditor/auditor.py

```diff
@@ -0,0 +1,163 @@
+        if not file_content:
+            logger.debug("File not found")
+            continue
+
+        if not audit_results["Uses CodeQL"]:
+            audit_results["Uses CodeQL"] = "uses: github/codeql-action/analyze" in file_content
+
+        if not audit_results["Uses Allowed list"]:
+            audit_results["Uses Allowed list"] = "uses: ed-fi-alliance-oss/ed-fi-actions/.github/workflows/repository-scanner.yml" in file_content
+
+        if not audit_results["Uses Test Reporter"]:
+            audit_results["Uses Test Reporter"] = "uses: dorny/test-reporter" in file_content
+
+        if not audit_results["Has Unit Tests"]:
+            pattern = re.compile(r"unit.{0,2}test(s)?", flags=re.IGNORECASE)
+            audit_results["Has Unit Tests"] = True if pattern.search(file_content) else False
+
+        if not audit_results["Has Linter"]:
+            pattern = re.compile(r"lint(er)?(s)?(ing)?", flags=re.IGNORECASE)
+            audit_results["Has Linter"] = True if pattern.search(file_content) else False
+
+    return audit_results
+
+
+def get_repo_information(client: GitHubClient, organization: str, repository: str) -> dict:
+    information = client.get_repository_information(organization, repository)
+
+    # Currently,this is only checking if there are alerts, which does not differentiates if dependabot is enabled or not
+    vulnerabilities = [alerts for alerts in information["vulnerabilityAlerts"]["nodes"]
+                       if (alerts["createdAt"] < (datetime.now() - timedelta(ALERTS_WEEKS_SINCE_CREATED * 7)).isoformat() and
+                       alerts["securityVulnerability"]["advisory"]["severity"] in ALERTS_INCLUDED_SEVERITIES)]
+
+    rulesForMain = [rules for rules in information["branchProtectionRules"]["nodes"] if rules["pattern"] == "main"]
+    rules = rulesForMain[0] if rulesForMain else None
+
+    logger.debug(f"Repository information: {information}")
+
+    return {
+        "Requires Signed commits": rules["requiresCommitSignatures"] if rules else False,
+        "Requires Code review": rules["requiresApprovingReviews"] if rules else False,
+        "Requires PR": (rules["requiresApprovingReviews"] and rules["isAdminEnforced"]) if rules else False,
+        "Has Wiki Enabled": information["hasWikiEnabled"],
+        "Has Issues Enabled": information["hasIssuesEnabled"],
+        "Has Projects Enabled": information["hasProjectsEnabled"],
+        "Has Discussions": information["discussions"]["totalCount"] > 0,
+        "Deletes head branch": information["deleteBranchOnMerge"],
+        "Uses Squash Merge": information["squashMergeAllowed"],
+        "Has License Information": information["licenseInfo"] is not None,
+        "Has Dependabot alerts": len(vulnerabilities) > 0
+    }
+
+
+def review_files(client: GitHubClient, organization: str, repository: str) -> dict:
+    files = {
+        "README.md": False,
+        "CONTRIBUTORS.md": False,
+        "NOTICES.md": False,
+        "LICENSE": False
```

Comment:
Some of our repositories have a LICENSE.txt file instead. Can you add logic to detect either LICENSE or LICENSE.txt?